src/deploy/functions/runtimes/discovery/v1alpha1.ts

```diff
@@ -4,6 +4,8 @@ import { copyIfPresent } from "../../../../gcp/proto";
 import { assertKeyTypes, requireKeys } from "./parsing";
 import { FirebaseError } from "../../../../error";
 
+const CHANNEL_NAME_REGEX = /^(projects\/([^/]+)\/)?locations\/([^/]+)\/channels\/([^/]+)$/;
```

Comment:
nit* We found that using named capturing group to help with readability of regex parsing, e.g.

https://github.com/firebase/firebase-tools/blob/19d3e24b488b4b96bc4428284648f17404b0146d/src/gcp/secretManager.ts#L9-L14

(Although it only really helps with readability if you know about the feature lol).

It feels slightly less brittle than using `match[0]` indexing imo.

Reply:
Done.